handlers/create_task.go

```diff
@@ -38,19 +41,26 @@ func CreateTask(ts database.TaskStore) http.HandlerFunc {
 
 		id, err := ts.CreateTask(task)
 		if err != nil {
-			http.Error(writer, errToJSON(err), http.StatusBadRequest)
+			log.Printf(err.Error())
+			err = errors.New(clientCreateErr)
+			http.Error(writer, errToJSON(err), http.StatusInternalServerError)
 			return
 		}
 
 		response, err := json.Marshal(TaskID{ID: id})
 		if err != nil {
-			http.Error(writer, errToJSON(err), http.StatusBadRequest)
+			log.Printf(err.Error())
+			err = errors.New(clientCreateErr)
+			http.Error(writer, errToJSON(err), http.StatusInternalServerError)
 			return
 		}
 
 		writer.Header().Set("Content-Type", "application/json; charset=UTF-8")
 		writer.WriteHeader(http.StatusCreated)
-		_, _ = writer.Write(response)
+		_, err = writer.Write(response)
+		if err != nil {
+			log.Printf(err.Error())
```

Comment:
Лучше всегда писать  небольшой комментарий, чтобы лог проще было читать
```suggestion
			log.Printf("error while writing request in CreateTask: %v, err)
```